src/mgmt/cli_mgmt.c

```diff
@@ -644,14 +644,15 @@ dc_mgmt_net_cfg(const char *name, crt_init_options_t *crt_info)
 
 	d_getenv_bool("D_DYNAMIC_CTX", &d_dynamic_ctx_g);
 	if (d_dynamic_ctx_g) {
-		int i;
+		int i, cnt = 0;
```

Comment:
style nit - our C style guide says each variable should be on its own line, even when they are the same type.